### ISSUE TITLE
`plugin:svelte/all` not working

### ISSUE THREAD

**Issue by devunt** [sveltejs/eslint-plugin-svelte#510]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.43.0

### What version of `eslint-plugin-svelte` are you using?

2.31.0

### What did you do?

```js
      files: ['src/**/*.svelte'],
      parser: 'svelte-eslint-parser',
      parserOptions: { parser: '@typescript-eslint/parser' },
      extends: ['plugin:svelte/all', 'plugin:svelte/prettier'],
```

### What did you expect to happen?

Works well

### What actually happened?

```
Error: .eslintrc.cjs#overrides[1] » plugin:svelte/all:
	Configuration for rule "svelte/no-restricted-html-elements" is invalid:
	Value [] should NOT have fewer than 1 items.
```

### Link to **GitHub Repo** with Minimal Reproducible Example

Not needed

### Additional comments

Tried to add `'svelte/no-restricted-html-elements': 'off'` to work around, but it did not work.
